cli/command/image/build.go

```diff
@@ -115,6 +116,7 @@ func NewBuildCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.StringVar(&options.networkMode, "network", "default", "Set the networking mode for the RUN instructions during build")
 	flags.SetAnnotation("network", "version", []string{"1.25"})
 	flags.Var(&options.extraHosts, "add-host", "Add a custom host-to-IP mapping (host:ip)")
+	flags.StringVar(&options.target, "target", "", "Set the target build stage to build.")
```

Comment:
The current wording is just a bit ambiguous. On first reading I thought it was saying 'This flags sets the target build stage to the value of "build"'.

That doesn't really tell the user how the behaviour will change (target is what gets tagged, some context don't build, etc).

Maybe something like: "Build stage to build and optionally tag", "Build target build stage", or something like that?